File: src/main/java/miragefairy2019/mod/lib/OreIngredientComplex.java
package miragefairy2019.mod.lib;

import net.minecraftforge.oredict.OreIngredient;

public class OreIngredientComplex extends OreIngredient
{

	public OreIngredientComplex(String ore)
	{
		super(ore);
	}

	@Override
	public boolean isSimple()
	{
		// これがtrueになっていると、subItemsを参照するようになるのでクラフティングツールが反応しなくなる
		return false;
	}

}

File: src/main/java/miragefairy2019/libkt/OreIngredientComplex.kt
package miragefairy2019.libkt

import net.minecraftforge.oredict.OreIngredient

class OreIngredientComplex(ore: String) : OreIngredient(ore) {
    // これがtrueになっていると、subItemsを参照するようになるのでクラフティングツールが反応しなくなる
    override fun isSimple(): Boolean = false
}
